src/libraries.sh

```diff
@@ -487,6 +488,15 @@ rlImport() {
     if ! eval $VERIFIER
     then
       rlLogError "rlImport: Import of library $LIBRARY was not successful (callback failed)"
+      if ! declare -F $VERIFIER > /dev/null
+      then
+        rlLogInfo "rlImport: Callback function $VERIFIER does not exist (see lib.sh: '#   library-prefix = $PREFIX')"
```

Comment:
just an indentation to better visualize the relation to the previous error
```suggestion
        rlLogInfo "rlImport:   Callback function $VERIFIER does not exist (see lib.sh: '#   library-prefix = $PREFIX')"
```